clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -25,10 +25,9 @@ object ClioClient extends App {
 
 class ClioClient(val webClient: ClioWebClient) {
 
-  val parser = new BaseParser
-
   def execute(args: Array[String])(implicit ec: ExecutionContext): Int = {
-    val success = parser.parse(args, BaseArgs()) match {
+    val parser: BaseParser = new BaseParser
```

Comment:
So you have to create a new parser every time you want to call `parse()`? It looks like the way you implemented `BaseParser` results in an immutable scopt parser.